### ISSUE TITLE
AttributeError: 'list' object has no attribute 'lower'

### ISSUE THREAD

**Issue by pablobh** [Diaoul/subliminal#917]
Every now and then I get this error on some files:
```
Traceback (most recent call last):
  File "/usr/local/bin/subliminal", line 9, in <module>
    load_entry_point('subliminal==2.1.0.dev0', 'console_scripts', 'subliminal')()
  File "/Library/Python/2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Library/Python/2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Python/2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Python/2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click-6.7-py2.7.egg/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "build/bdist.macosx-10.13-intel/egg/subliminal/cli.py", line 400, in download
  File "build/bdist.macosx-10.13-intel/egg/subliminal/core.py", line 261, in download_best_subtitles
  File "build/bdist.macosx-10.13-intel/egg/subliminal/score.py", line 106, in compute_score
  File "build/bdist.macosx-10.13-intel/egg/subliminal/providers/opensubtitles.py", line 102, in get_matches
  File "build/bdist.macosx-10.13-intel/egg/subliminal/subtitle.py", line 241, in guess_matches
AttributeError: 'list' object has no attribute 'lower'
```

I tried running the develop and the master branches with the same error. Any ideas on what can be done to fix this?

**Comment by riramar** [comment]
Same problem here with the latest version.

**Comment by bibiak1** [comment]
Fit it with this line:
if video.source and 'source' in guess and [x for x in guess['source'] if x.lower() == video.source.lower()]:

**Comment by ratoaq2** [comment]
Fixed by 3121a75499779c7cc9eba059a4fa123cf5541027

**Comment by CWempe** [comment]
Tried the updated docker image (latest).

Seems to work fine.

Thanks
